src/components/navigation/header/config/HeaderNavigationConfig.tsx

```diff
@@ -14,7 +16,7 @@ import {
 } from 'src/components/navigation/link';
 import { BaseConfig } from 'src/components/navigation/BaseConfig';
 
-import { User, UserType } from '../types';
```

Comment:
Maybe we can remove `User` defined here in types, since we are using now `MergedUser` type from auth?